lib/components/artwork_grids/generic_grid.js

```diff
@@ -129,6 +129,8 @@ const styles = StyleSheet.create({
 })
 
 const GenericArtworksGridContainer = Relay.createContainer(GenericArtworksGrid, {
+  shouldComponentUpdate: () => true,
```

Comment:
I was hopeful about this, but it doesn't seem to make a difference. Temporarily leaving it here in the PR so others don't go down the same rabbit hole.